Analysis/output/country_data_report.Rmd

```diff
@@ -92,64 +101,62 @@ taxdat::plot_raw_observations(disjoint_set_sf_cases,
 ## Covariates
 
 ### Population
-```{r pop, echo=FALSE, message=FALSE, warning=FALSE, fig.height=5, fig.width=10, fig.cap="Population density"}
-taxdat::plot_raster_population(covar_data_filename = file_names["covar"], 
-                               render = T)
+```{r pop, fig.height=5, fig.width=10, fig.cap="Population density"}
+taxdat::plot_raster_population(covar_data_filename = file_names["covar"], render = T)
 ```
 
 ### Other covariates
 
-```{r covars, echo=FALSE, message=FALSE, warning=FALSE, fig.height=5, fig.width=8, fig.cap="Covariate rasters"}
-taxdat::plot_raster_covariates(covar_data_filename = file_names["covar"], 
-                               render = T)
+```{r covars, fig.height=5, fig.width=8, fig.cap="Covariate rasters"}
+taxdat::plot_raster_covariates(covar_data_filename = file_names["covar"], render = T)
 ```
 
 ## Output maps
 
-```{r caserast, echo=FALSE, fig.cap="Modeled cases", fig.height=5, fig.width=10, message=FALSE, warning=FALSE}
-case_raster <- taxdat::get_case_raster(preprocessed_data_filename = file_names["data"],
+```{r caserast, fig.cap="Modeled cases", fig.height=5, fig.width=10}
+case_raster <- taxdat::get_case_raster(preprocessed_data_filename = preprocessed_data_filename,
                                        covar_data_filename = file_names["covar"],
```

Comment:
If we're replacing file_names["data"] everywhere, we should also replace all of the other `file_names` instances. My preference is to use file_names everywhere and not create the preprocessed_data_filename etc, but don't care too much

Reply:
agree